platform-includes/getting-started-config/javascript.nuxt.mdx

```diff
@@ -44,40 +44,33 @@ Sentry.init({
 import * as Sentry from '@sentry/nuxt';
 
 Sentry.init({
-  dsn: "___PUBLIC_DSN___"
+  dsn: "___PUBLIC_DSN___",
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0
 });
 ```
 
 The Nuxt `useRuntimeConfig()` does not work in the Sentry server config due to technical reasons (the config file has to
 be loaded before Nuxt is loaded). To be able to use `process.env` you either have to add `--env-file=.env` to your node command
-or use the `dotenv` package:
 
-```bash {tabTitle: env-file}
-node --env-file=.env --import ./.output/server/sentry.server.config.mjs .output/server/index.mjs
+```bash {tabTitle: node}
+node --env-file=.env .output/server/index.mjs
 ```
 
+or use the `dotenv` package:
 
-
-```javascript {tabTitle: dotenv} {filename:sentry.server.config.ts} {1,4}
+```javascript {tabTitle: Server Config} {filename:sentry.server.config.ts} {1,3}
 import dotenv from 'dotenv';
-import * as Sentry from '@sentry/nuxt';
 
 dotenv.config();
 
-Sentry.init({
-  dsn: "___PUBLIC_DSN___"
-});
+// ... rest of the file
 ```
```

Comment:
No need to act now - just sharing my thoughts: I don't think how users can pass env vars into their app shouldn't be in our documentation but idk.

Reply:
I added it because there's no need to do those extra steps when using Nuxt. Usually, you can use `useRuntimeConfig()`, where your env vars are set and `process.env` can be used without explicitly installing `dotenv`. 

However, as the server-side config is loaded before any Nuxt code is loaded, the env setup has to be done manually.